static/app/components/devtoolbar/styles/reset.ts

```diff
@@ -15,6 +15,12 @@ export const resetFlexColumnCss = css`
   flex-grow: 1;
 `;
 
+export const resetFlexRowCss = css`
+  display: flex;
+  flex-direction: row;
```

Comment:
it's true!

Reply:
we need this because i was messing around and added this global:

https://github.com/getsentry/sentry/blob/8cfb85f3b9759e7486020924c6533fb281260184/static/app/components/devtoolbar/styles/global.ts#L57-L60

Reply:
ahh i see